dali/kernels/imgproc/warp_cpu.h

```diff
@@ -85,38 +87,56 @@ class WarpCPU {
   }
 
  private:
-  template <DALIInterpType static_interp>
+  template <DALIInterpType static_interp, typename Mapping_>
```

Comment:
why not `Mapping` (no underscore) ?

Reply:
Because when Mapping is AffineMapping2D, then we've got multiple function definition.